src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
If Flow had opaque type aliases and we used them here (which I'd like to do), this wouldn't typecheck. Use ReactPriorityLevel.NoWork instead?